ansible/edpm_provisioning.yml

```diff
@@ -0,0 +1,204 @@
+---
+- name: Provisioning EDPM nodes
+  hosts: localhost
+  gather_facts: true
+
+  tasks:
+    - name: Download OCP inventory file (scalelab)
```

Comment:
Can you reuse bootstrap role for downloading inventory?
Or include tasks like below, if this role is intended to be independant of RHOSO deployment flow https://github.com/redhat-performance/JetBrew/blob/main/ansible/deploy_external_ceph.yaml#L7-L8